client/autoclient.go

```diff
@@ -0,0 +1,452 @@
+package client
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"sync"
+
+	"decred.org/dcrwallet/v3/errors"
+	"decred.org/dcrwallet/v3/wallet"
+	"decred.org/dcrwallet/v3/wallet/udb"
```

Comment:
Could this and the feepayment file just use "decred.org/dcrwallet/v2" for now? v3 is not yet public, so it made sense to use when this was all internal, but now the consumer will have to replace for v3 modules in order to use the `Wallet` interface. I have confirmed v2 works still...